scripts/artifacts-building/apt/aptly-skip-vars.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
We can remove this file if we include aptly_dont_snapshot_list: [] into the aptly-vars.yml
More readable.